angular-ngrx-scss/src/app/file-viewer/components/file-explorer-about/file-explorer-about.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container border-bottom-2">
+  <h2>About</h2>
+  <div class="description mt-1">
+    <span data-testid="repo file explorer description">
+      {{ description ?? 'No description, website, or topics provided.' }}
+    </span>
+
+    <div *ngIf="homepageUrl" class="linkContainer mt-1">
+      <span class="icon" appOcticon="link"></span>
+      <a [href]="homepageUrl" class="link" target="_blank">
+        {{ homepageUrl }}
+      </a>
+    </div>
+
+    <ng-container *ngIf="topics">
+      <div class="mt-1">
+        <span *ngFor="let topic of topics" class="topic">{{ topic }}</span>
+      </div>
+    </ng-container>
+  </div>
+
+  <div class="mt-1">
+    <!--    TODO: this should scroll down to the readme section of the repository when it is clicked-->
```

Comment:
TODO's tend to get lost in the code. Be sure to add these to a new issue if not created already.